gcovr/writer/json.py

```diff
@@ -21,8 +21,14 @@
 import functools
```

Comment:
Once a file is reformatted with black, it should be added to the known-good list of files in the Noxfile.

Reply:
Of course.